bob-common/mkosi.extra/etc/fluent-bit/delay.lua

```diff
@@ -2,7 +2,7 @@
 -- Lua script stays in memory for the lifetime of the Fluent Bit process,
 -- so all local variables persist across filter calls.
 
-local DELAY_SEC = 300 -- Delay (in seconds) before flushing logs
+local DELAY_SEC = 30 -- Delay (in seconds) before flushing logs
```

Comment:
Is that intended change? I'll cherry-pick other two hunks into #43

Reply:
definitely not intended, lol